features/google/src/components/importExplanationExpandable/importExplanationExpandable.css

```diff
@@ -125,23 +125,15 @@ h5 {
 .import-view .explanation-expandable .section-body h4 {
     font-family: "Jost Medium";
     font-size: var(--font-size-xl);
-    margin-bottom: 36px;
-    margin-top: 20px;
+    margin-bottom: calc(var(--gdu) * 9);
+    margin-top: calc(var(--gdu) * 5);
 }
 
 .import-view .explanation-expandable .section-body h5 {
     font-weight: 500;
     font-size: var(--font-size-l);
-    margin-top: 10px;
-    margin-bottom: 10px;
-}
-
-.import-view .explanation-expandable .section-body .btn-1 {
```

Comment:
This deletion and the one above seem unrelated, however, I guess it's simply a path what was no longer used, right?

Reply:
We are no longer using it. But you're right, it's unrelated to this PR. If you think it's clearer, I can add it back and clean it in a different branch.

Reply:
No big deal. Let's just get rid of it.